pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -583,3 +614,40 @@ exit 0`,
 	}
 	return extraInitContainersMap, nil
 }
+
+func copyConfig(comp *vshnv1.VSHNKeycloak, svc *runtime.ServiceRuntime) error {
```

Comment:
I don't understand what is happening in this go function, you are setting 2 objects in the desired function and then getting from observe, can you briefly explain why?

Reply:
I'm observing the configmap in the claim, to be able to copy the content of the configmap to a new configmap in the instance namespace